src/main/charts/bamboo/values.yaml

```diff
@@ -2,6 +2,14 @@
 # 
 # HEADS UP! 
 #
+# Release 1.0.0 of this chart only support FULL unattended setups. 
```

Comment:
So it is not possible to have either an unattended setup or a fully manual setup? I thought we can give the choice to the customers.

Reply:
This will affect terraform project as well. What we are going to do in terraform part is make sure the user has provided all info for unattended setup otherwise we stop them to proceed. 

Do we need to give users choice to do manual setup?

Reply:
During testing full manual setups at times [exhibited issues](https://hello.atlassian.net/wiki/spaces/DCD/pages/1450841428/Fix+verification+-+testing#Nothing-pre-seeded---working-but-flakey)

Because of this Im not comfortable suggesting this as an approach right now.